Mon Jun 16 16:12:18 CEST 2003	Arnaud Charlet	<charlet@act-europe.fr>

	glade/main.c: Disable gail and non portable nls calls.
	glade/glade_project.c: Set Ada as the default language.
	configure.in: Disable check for scrollkeeper, not needed/available.

Thu Jun 12 12:05:57 CEST 2003	Nicolas Setton	<setton@act-europe.fr>

	glade/glade_project.c: Generate Ada as well as C and C++.

--- glade/main.c.old	Mon Jun 16 15:53:41 2003
+++ glade/main.c	Mon Jun 16 16:07:16 2003
@@ -122,7 +122,7 @@
 #ifdef USE_GNOME
   modules_needed = "gail"; /* FIXME: Add gail-gnome when it is ready. */
 #else
-  modules_needed = "gail";
+  modules_needed = "";
 #endif
   new_modules = g_strdup_printf ("GTK_MODULES=%s%s%s",
 				 modules ? modules : "",
@@ -173,7 +173,7 @@
 
 #ifdef ENABLE_NLS
   bindtextdomain (GETTEXT_PACKAGE, GLADE_LOCALEDIR);
-  bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
+  /* bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8"); */
   textdomain (GETTEXT_PACKAGE);
 #endif
 
--- glade/glade_project.c.old	Mon Jun 16 16:02:15 2003
+++ glade/glade_project.c	Mon Jun 16 16:03:28 2003
@@ -46,7 +46,7 @@
 /* The order must match the GladeLanguageType enum in glade_project.h. */
 gchar *GladeLanguages[] = { "C", "C++" , "Ada 95", "Perl", "Eiffel" };
 #if 1
-gint GladeNumLanguages = 2; /* Only C and C++ for now. */
+gint GladeNumLanguages = 3; /* Only C and C++ for now. */
 #else
 gint GladeNumLanguages = sizeof (GladeLanguages) / sizeof (GladeLanguages[0]);
 #endif
@@ -160,7 +160,7 @@
   project->source_directory = NULL;
   project->pixmaps_directory = NULL;
 
-  project->language = GLADE_LANGUAGE_C;
+  project->language = GLADE_LANGUAGE_ADA95;
   property_show_lang_specific_page (project->language);
 
   project->changed = FALSE;
--- configure.in.old	Mon Jun 16 15:45:38 2003
+++ configure.in	Mon Jun 16 15:45:39 2003
@@ -114,18 +114,6 @@
 
 
 dnl ***************************************************************************
-dnl Check for ScrollKeeper
-dnl ***************************************************************************
-
-AC_PATH_PROG(SCROLLKEEPER_CONFIG, scrollkeeper-config,no)
-if test "$SCROLLKEEPER_CONFIG" = "no"; then
-  AC_MSG_ERROR(Couldn't find scrollkeeper-config, please install the scrollkeeper package)
-fi
-SCROLLKEEPER_REQUIRED=0.1.4
-AC_SUBST(SCROLLKEEPER_REQUIRED)
-
-
-dnl ***************************************************************************
 dnl Debugging - defines GLADE_DEBUG if --enable-debug option is used.
 dnl ***************************************************************************
 
--- glade/main.c.old	Mon Jun 16 15:53:41 2003
+++ glade/main.c	Mon Jun 16 16:07:16 2003
@@ -122,7 +122,7 @@
 #ifdef USE_GNOME
   modules_needed = "gail"; /* FIXME: Add gail-gnome when it is ready. */
 #else
-  modules_needed = "gail";
+  modules_needed = "";
 #endif
   new_modules = g_strdup_printf ("GTK_MODULES=%s%s%s",
 				 modules ? modules : "",
@@ -173,7 +173,7 @@
 
 #ifdef ENABLE_NLS
   bindtextdomain (GETTEXT_PACKAGE, GLADE_LOCALEDIR);
-  bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
+  /* bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8"); */
   textdomain (GETTEXT_PACKAGE);
 #endif
 
--- glade/glade_project.c.old	Mon Jun 16 16:02:15 2003
+++ glade/glade_project.c	Mon Jun 16 16:11:07 2003
@@ -46,7 +46,7 @@
 /* The order must match the GladeLanguageType enum in glade_project.h. */
 gchar *GladeLanguages[] = { "C", "C++" , "Ada 95", "Perl", "Eiffel" };
 #if 1
-gint GladeNumLanguages = 2; /* Only C and C++ for now. */
+gint GladeNumLanguages = 3; /* Only C, C++ and Ada for now. */
 #else
 gint GladeNumLanguages = sizeof (GladeLanguages) / sizeof (GladeLanguages[0]);
 #endif
@@ -160,7 +160,7 @@
   project->source_directory = NULL;
   project->pixmaps_directory = NULL;
 
-  project->language = GLADE_LANGUAGE_C;
+  project->language = GLADE_LANGUAGE_ADA95;
   property_show_lang_specific_page (project->language);
 
   project->changed = FALSE;
--- configure.in.old	Mon Jun 16 15:45:38 2003
+++ configure.in	Mon Jun 16 15:45:39 2003
@@ -114,18 +114,6 @@
 
 
 dnl ***************************************************************************
-dnl Check for ScrollKeeper
-dnl ***************************************************************************
-
-AC_PATH_PROG(SCROLLKEEPER_CONFIG, scrollkeeper-config,no)
-if test "$SCROLLKEEPER_CONFIG" = "no"; then
-  AC_MSG_ERROR(Couldn't find scrollkeeper-config, please install the scrollkeeper package)
-fi
-SCROLLKEEPER_REQUIRED=0.1.4
-AC_SUBST(SCROLLKEEPER_REQUIRED)
-
-
-dnl ***************************************************************************
 dnl Debugging - defines GLADE_DEBUG if --enable-debug option is used.
 dnl ***************************************************************************
 
